modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionExchangeId.java

```diff
@@ -105,13 +104,27 @@ public UUID nodeId() {
         return nodeId;
     }
 
+    /**
+     * @param nodeId New node ID.
+     */
+    public void nodeId(UUID nodeId) {
+        this.nodeId = nodeId;
+    }
+
     /**
      * @return Event.
      */
     public int event() {
         return evt;
     }
 
+    /**
+     * @param evt New event type.
+     */
+    public void event(int evt) {
```

Comment:
I suggest to rename this method to `eventType`.

Reply:
It leads to renaming of getter too.